fineract-provider/src/main/java/org/apache/fineract/useradministration/starter/UserAdministrationAutoConfiguration.java

```diff
@@ -0,0 +1,126 @@
+import org.apache.fineract.organisation.office.service.OfficeReadPlatformService;
+import org.apache.fineract.organisation.staff.domain.StaffRepositoryWrapper;
+import org.apache.fineract.organisation.staff.service.StaffReadPlatformService;
+import org.apache.fineract.portfolio.client.domain.ClientRepositoryWrapper;
+import org.apache.fineract.useradministration.data.PasswordPreferencesDataValidator;
+import org.apache.fineract.useradministration.domain.AppUserPreviousPasswordRepository;
+import org.apache.fineract.useradministration.domain.AppUserRepository;
+import org.apache.fineract.useradministration.domain.PasswordValidationPolicyRepository;
+import org.apache.fineract.useradministration.domain.PermissionRepository;
+import org.apache.fineract.useradministration.domain.RoleRepository;
+import org.apache.fineract.useradministration.domain.UserDomainService;
+import org.apache.fineract.useradministration.serialization.PermissionsCommandFromApiJsonDeserializer;
+import org.apache.fineract.useradministration.service.AppUserReadPlatformService;
+import org.apache.fineract.useradministration.service.AppUserReadPlatformServiceImpl;
+import org.apache.fineract.useradministration.service.AppUserWritePlatformService;
+import org.apache.fineract.useradministration.service.AppUserWritePlatformServiceJpaRepositoryImpl;
+import org.apache.fineract.useradministration.service.PasswordPreferencesWritePlatformService;
+import org.apache.fineract.useradministration.service.PasswordPreferencesWritePlatformServiceJpaRepositoryImpl;
+import org.apache.fineract.useradministration.service.PasswordValidationPolicyReadPlatformService;
+import org.apache.fineract.useradministration.service.PasswordValidationPolicyReadPlatformServiceImpl;
+import org.apache.fineract.useradministration.service.PermissionReadPlatformService;
+import org.apache.fineract.useradministration.service.PermissionReadPlatformServiceImpl;
+import org.apache.fineract.useradministration.service.PermissionWritePlatformService;
+import org.apache.fineract.useradministration.service.PermissionWritePlatformServiceJpaRepositoryImpl;
+import org.apache.fineract.useradministration.service.RoleDataValidator;
+import org.apache.fineract.useradministration.service.RoleReadPlatformService;
+import org.apache.fineract.useradministration.service.RoleReadPlatformServiceImpl;
+import org.apache.fineract.useradministration.service.RoleWritePlatformService;
+import org.apache.fineract.useradministration.service.RoleWritePlatformServiceJpaRepositoryImpl;
+import org.apache.fineract.useradministration.service.UserDataValidator;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+public class UserAdministrationAutoConfiguration {
```

Comment:
This is not auto configuration. Please rename it.

Reply:
Done